lua/lv-which-key/init.lua

```diff
@@ -74,10 +71,24 @@ vim.api.nvim_set_keymap('n', '<leader>p', ":lua require'telescope'.extensions.pr
 local mappings = {
     ["/"] = "Comment",
     ["c"] = "Close Buffer",
-    ["e"] = "Explorer",
+    ["e"] = {"<cmd>lua require\"lv-nvimtree\".toggle_tree()<CR>", "Explorer"},
     ["f"] = "Find File",
     ["h"] = "No Highlight",
     ["p"] = "Projects",
+    ['<TAB>']   =  {'<cmd>BufferNext<cr>', 'jump to next buffer'},
+    ['<S-TAB>'] =  {'<cmd>BufferPrevious<cr>' , 'jump to prev buffer'},
+    ['<S-x>']   =  {'<cmd>BufferClose<cr>', 'close buffer'},
```

Comment:
There is already a buffer close binding, see ["c"]

Reply:
Interesting! I basically kept the same bindings from [lua/lv-barbar/init.lua](https://github.com/ChristianChiarulli/LunarVim/blob/19bd48ac809cf7e17ac5f56acf0b4153f311e7e1/lua/lv-barbar/init.lua). So I guess it was a duplicate all along. Not sure which one people use more often tho.